src/connector/codec/src/lib.rs

```diff
@@ -42,3 +42,16 @@
 /// Converts JSON/AVRO/Protobuf data to RisingWave datum.
 /// The core API is [`decoder::Access`].
 pub mod decoder;
+
+pub use risingwave_pb::plan_common::ColumnDesc;
+pub type RisingWaveSchema = Vec<ColumnDesc>;
```

Comment:
Does the name sound too general to the outsiders? 😄

Reply:
removed. Add later if needed